GBAP/GBAPModel.cpp

```diff
@@ -190,5 +193,16 @@ void GBAP::mult(std::vector<float>& arr, float scal)
   for(float& f : arr)
     f *= scal;
 }
-}
 
+void GBAP::normalizeArray(std::vector<float>& arr){
+
+  float max = *std::max_element(arr.begin(), arr.end());
```

Comment:
Here we should test that arr is not empty() before doing anything otherwise it will crash